src/OrbitBase/include/OrbitBase/Profiling.h

```diff
@@ -51,7 +51,7 @@ namespace orbit_base {
 constexpr clockid_t kOrbitCaptureClock = CLOCK_MONOTONIC;
 
 [[nodiscard]] inline uint64_t CaptureTimestampNs() {
-  timespec ts;
+  timespec ts{};
```

Comment:
This also seems uncalled for as this is an output parameter of `clock_gettime` that follows.

Reply:
Done.